tests/e2e/test_rosacli_upgrade.go

```diff
@@ -483,7 +478,7 @@ var _ = Describe("Describe/List rosa upgrade",
 					By("Check list upgrade")
 					out, err := upgradeService.ListUpgrades(clusterID)
 					Expect(err).To(BeNil())
-					Expect(out.String()).To(ContainSubstring("%s   recommended - scheduled for %s %s UTC", upgradingVersion,
+					Expect(out.String()).To(ContainSubstring("%s  recommended - scheduled for %s %s UTC", upgradingVersion,
```

Comment:
Better to use regex to match in case of future fixing.

Reply:
Updated PR according to comment.